src/Selenium2Library/keywords/_tableelement.py

```diff
@@ -73,6 +73,37 @@ def table_cell_should_contain(self, table_locator, row, column, expected, loglev
             self.log_source(loglevel)
             raise AssertionError(message)
 
+    def table_cell_should_not_contain(self, table_locator, row, column, expected, loglevel='INFO'):
```

Comment:
Perhaps `expected` is not good argument name, because we expect that string is not there. Could `not_contain` or `value` be better argument names.